llvm/test/CodeGen/X86/insert.ll

```diff
@@ -0,0 +1,93 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -mtriple=i386-unknown-unknown | FileCheck %s --check-prefixes=I386
+; RUN: llc < %s -mtriple=x86_64-unknown-unknown | FileCheck %s --check-prefixes=X86_64
```

Comment:
(style) We try to use 'X86' for 32-bit triple checks and 'X64' for 64-bit triple checks.

Reply:
Fixed.